tests/validation/tests/v3_api/test_windows_cluster.py

```diff
@@ -65,3 +57,26 @@ def pull_images(node):
                                        + " && " +
                                        "docker pull " + TEST_IMAGE_OS_BASE)
     print(pull_result)
+
+
+def create_windows_vxlan(version=K8S_VERSION):
```

Comment:
These two methods have pretty much the same logic except the `disable_source_dest_check`, wouldn't it better be parameterized ? 

`create_windows_cluster(network_type, k8s_version)`

Reply:
you are right. It can be done. I will look it to it. thank you